apps/src/code-studio/hiddenLessonRedux.js

```diff
@@ -195,7 +195,7 @@ export function getHiddenLessons(scriptName, canHideLessons) {
   return dispatch => {
     $.ajax({
       type: 'GET',
-      url: `/s/${scriptName}/hidden_stages`,
+      url: `/s/${scriptName}/hidden_lessons`,
```

Comment:
When I go to /s/csd2-2021/hidden_lessons on production I get a 404 sad bee whereas /s/csd2-2021/hidden_stages works. I think the routes need to be updated?

Reply:
On production or local?

Reply:
🤦‍♀️ I looked for the change in this PR and didn't see it but it's clearly there

Reply:
This is what I get for reviewing PRs early in the morning :)